xinference/types.py

```diff
@@ -300,6 +300,16 @@ class PytorchGenerateConfig(TypedDict, total=False):
     lora_name: Optional[str]
     stream_options: Optional[Union[dict, None]]
     request_id: Optional[str]
+    platform: Optional[Literal["Mac", "WIN", "Mobile"]]
```

Comment:
Looks like this is for cogagent only, is it possible to put them into `**kwargs` stuff?